scripts/install

```diff
@@ -103,6 +103,11 @@ ynh_add_nginx_config
 #=================================================
 ynh_script_progression --message="Configuring a systemd service..." --weight=1
 
+if [ ! -d directory ]; then
+	mkdir /var/log/13ft
+fi
```

Comment:
Eeeeh you are testing if a folder literally named `directory` exists, and otherwise creating `/var/log/13ft` .. I uuuuh ... what, why :sweat_smile:

Reply:
:rofl: copy pasting commands without changing them :joy: